src/euphorie/client/browser/templates/sessions-dashboard.pt

```diff
@@ -107,13 +107,27 @@
              data-pat-masonry="column-width: .grid-sizer; gutter: 0; item-selector: .portlet; stamp: .stamp;"
         >
           <div class="grid-sizer"></div>
-          <tal:portlets repeat="portlet view/portlets"
-                        replace="structure portlet"
-          />
+          <tal:portlets repeat="portlet view/portlets">
+            <div class="portlet span-${portlet/columns|string:1}"
+                 id="${portlet/element_id}"
+                 tal:condition="portlet/element_id|nothing"
+            >
+              <a class="pat-inject infinite-scrolling-trigger"
+                 href="${context/absolute_url}/@@${portlet/__name__}#${portlet/element_id}"
+                 data-pat-inject="trigger: autoload"
```

Comment:
I wonder if `trigger: autoload-visible` should be used here. This adds a small JavaScript overhead though.